### ISSUE TITLE
Nightly build has been marked as latest release

### ISSUE THREAD

**Issue by rainyclouds** [brave/brave-browser#27195]
## Description 
The https://github.com/brave/brave-browser/releases/tag/v1.48.32 release has been marked as latest. I believe this is incorrect and the latest release should perhaps be https://github.com/brave/brave-browser/releases/tag/v1.46.134

This will affect anyone using the https://github.com/brave/brave-browser/releases/latest URL to download the latest release.

## Steps to Reproduce

   1. Visit https://github.com/brave/brave-browser/releases/latest

## Actual result:
Visiting the https://github.com/brave/brave-browser/releases/latest URL links to https://github.com/brave/brave-browser/releases/tag/v1.48.32

## Expected result:
Visiting the https://github.com/brave/brave-browser/releases/latest URL should link to https://github.com/brave/brave-browser/releases/tag/v1.46.134

## Reproduces how often: 
Every time as of a recent nightly build.

**Comment by rebron** [comment]
cc: @mihaiplesa

**Comment by mihaiplesa** [comment]
Yes, we're fixing in https://github.com/brave/devops/issues/8870
